src/styles/_navigation.scss

```
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+  &__logo {
```

```suggestion
  height: 60px;
  
  &__logo {
```
Please, apply all the changes that the previous mentors have requested.